### PR TITLE
Port sky viewer to new view manager.

### PR DESCRIPTION
**Pull request by j9brown**
Requires a fresh roll from Mojo repo before submitting.

### CODE REVIEW

File: services/sky/content_handler_impl.cc [hunk 1]
```diff
@@ -5,14 +5,57 @@
...
+#include "mojo/public/cpp/application/service_provider_impl.h"
 #include "mojo/public/cpp/bindings/strong_binding.h"
 #include "mojo/public/cpp/utility/run_loop.h"
 #include "mojo/services/network/interfaces/network_service.mojom.h"
+#include "mojo/services/ui/views/interfaces/view_provider.mojom.h"
 #include "services/sky/document_view.h"
 
 namespace sky {
 
+class SkyViewProvider : public mojo::InterfaceFactory<mojo::ui::ViewProvider>,
```
Comment (abarth) [inline services/sky/content_handler_impl.cc]: FlutterViewProvider ?

Comment (j9brown) [inline services/sky/content_handler_impl.cc]: Haha!  Let's refactor that next.  :)

**Comment by abarth** [comment]
This looks great.  LGTM.

At some point, once the dust has settled on the new view manager, I'd like to refactor/rewrite this code.  It should be possible to delete all this code and integrate mojo with the `//sky/shell` embedding of `//sky/engine`.  Essentially, we should be able to treat mojo as just another host OS for the flutter shell.  That's probably the time to rename all this stuff to flutter, and we'll end up with a `flutter_shell.mojo`.

**Comment by j9brown** [comment]
Yup.  Sounds good to me.

**Pull request closed without merging**